### ISSUE TITLE
aws_waf_web_acl import not supported

### ISSUE THREAD

**Issue by bacoboy** [terraform-providers/terraform-provider-aws#4589]
Trying to import some stuff I setup manually into terraform only to discover there is no import for an existing waf_web_acl.  Expected this to work:
```
$ aws waf list-web-acls
{
    "WebACLs": [
        {
            "WebACLId": "7ef494fc-fa3f-77e1-a74e-2fa3f2e96c37",
            "Name": "mywaf"
        }
    ]
}
$ terraform plan
  + aws_waf_web_acl.waf
      id:                             <computed>
      default_action.#:               "1"
      default_action.4234791575.type: "ALLOW"
      metric_name:                    "mywaf"
      name:                           "mywaf"
$ terraform import aws_waf_web_acl.waf 7ef494fc-fa3f-77e1-a74e-2fa3f2e96c37
aws_waf_web_acl.waf: Importing from ID "7ef494fc-fa3f-77e1-a74e-2fa3f2e96c37"...
Error importing: 1 error(s) occurred:

* aws_waf_web_acl.waf (import id: 7ef494fc-fa3f-77e1-a74e-2fa3f2e96c37): import aws_waf_web_acl.waf (id: 7ef494fc-fa3f-77e1-a74e-2fa3f2e96c37): resource aws_waf_web_acl doesn't support import
```

**Comment by bflad** [comment]
The initial resource import implementation has been merged which handles everything but the `rules` associations and will be released with version 1.29.0 of the AWS provider. We'll circle back with finishing the `rules` implementation shortly.

**Comment by bflad** [comment]
Followup PR for `rules` handling: #5342
